guidelines/README.md

```diff
@@ -0,0 +1,5 @@
+# Guidelines
+
+The objective of guidelines is to share knowledge between club leaders in a more
```

Comment:
I wouldn't understand this either, or at least wouldn't be able to contribute to it...